src/utils/urlParseHelper.ts

```diff
@@ -15,15 +15,20 @@ export interface IParsedUrl {
  * @returns Object containing scmRepository URL, parsed URL and display name representing URL
  *  */
 export const parseInternalScmRepositoryUrl = ({ url }: IScmRepositoryUrl): IParsedUrl => {
-  const protocol = url.includes('git@') ? 'git@' : url.split('://')[0];
-  const base = url.includes('git@') ? url.split('@').at(1)?.split(':').at(0) : url.split('://')[1].split('/')[0];
-  const project = url.includes('git@')
+  const gitlabBase = 'gitlab.cee.redhat.com';
+  const isGitProtocol = url.includes('git@');
+  const protocol = isGitProtocol ? 'git@' : url.split('://')[0];
+  const base = isGitProtocol ? url.split('@').at(1)?.split(':').at(0) : url.split('://')[1].split('/')[0];
+  const project = isGitProtocol
     ? url.split(':').at(1)
+    : url.includes(gitlabBase)
+    ? url.split(gitlabBase + '/').at(1)
     : url.split(base + (['https', 'http'].includes(protocol) ? '/gerrit/' : '/'))[1];
-  const webUrl = url.includes('git@')
-    ? 'https://' + base + '/' + project
-    : 'https://' + base + '/gerrit/gitweb?p=' + project + ';a=summary';
-  const name = url.includes('git@') ? 'GitLab' : 'Gerrit';
+  const webUrl =
+    isGitProtocol || url.includes(protocol + '://' + gitlabBase)
+      ? 'https://' + base + '/' + project
+      : 'https://' + base + '/gerrit/gitweb?p=' + project + ';a=summary';
+  const name = url.includes('git@') || url.includes(protocol + '://' + gitlabBase) ? 'GitLab' : 'Gerrit';
```

Comment:
`url.includes('git@') || url.includes(protocol + '://' + gitlabBase)`
is computed twice, move it into variable (isGitlabUrl)
also `url.includes(gitlabBase)` might be enough